tmt/package_managers/bootc.py

```diff
@@ -23,6 +23,40 @@
 LOCALHOST_BOOTC_IMAGE_PREFIX = "localhost/tmt"
 
 
+class BootcMetadataContainer(MetadataContainer):
+    """
+    Metadata container for bootc images.
+    Derived from https://bootc-dev.github.io/bootc/host-v1.schema.json.
```

Comment:
Well the container is not derived from it. The schema is just a reference here. Also add a note that the container is intentionally left incomplete.